libs/ui/select/select.stories.ts

```diff
@@ -101,6 +105,76 @@ export const ReactiveFormControl: Story = {
 	}),
 };
 
+export const DisabledOption: Story = {
+	render: (args) => ({
+		props: { ...args, fruitGroup: new FormGroup({ fruit: new FormControl(args.initialValue) }) },
+		template: /* HTML */ `
+			<div class="mb-3">
+				<pre>Form Control Value: {{ fruitGroup.controls.fruit.value | json }}</pre>
+			</div>
+			<form [formGroup]="fruitGroup">
+				<brn-select
+					class="w-56"
+					${argsToTemplate(args, { exclude: ['initialValue', 'selectValueTransformFn'] })}
+					formControlName="fruit"
+				>
+					<hlm-select-trigger>
+						<brn-select-value hlm />
+					</hlm-select-trigger>
+					<hlm-select-content>
+						<hlm-select-label>Fruits</hlm-select-label>
+						<hlm-option value="apple">Apple</hlm-option>
+						<hlm-option data-testid="banana-option" value="banana" disabled>Banana</hlm-option>
+						<hlm-option value="blueberry">Blueberry</hlm-option>
+						<hlm-option value="grapes">Grapes</hlm-option>
+						<hlm-option value="pineapple">Pineapple</hlm-option>
+						<hlm-option>Clear</hlm-option>
+					</hlm-select-content>
+				</brn-select>
+				<form></form>
```

Comment:
```suggestion
				<form></form>
```
Is this on purpose?

Reply:
I just copied it from other stories. Did not even see the it ... ;) 
removed it.